### PR TITLE
Merge "sectors" into "topics"

### PR DESCRIPTION
**Pull request by tijmenb**
This PR is merges sectors into topics.

Currently the `/sectors` page duplicates the functionality of the `/topics` endpoint. This is confusing users as there are two almost identical pages with a list of topics.

This pull request:

- Removes the sectors page from the routes.
- Adds the topic children list to the topic index
- Adds a button to the curated list manage page at the topic page

screen shot 2015-05-15 at 12 43 54

This PR is part of a larger effort to make the UI for collections-publisher a bit nicer. A separate PR for that is currently under development, but I didn't want to do too much at once here. 

Trello: https://trello.com/c/u7tOFFXk

Best reviewed per-commit.

### CODE REVIEW

**Comment by tijmenb** [comment]
Comments from @alext:

- Only users with the 'GDS Editor' permission can create/edit topics, whereas all users can curate the contents of topics.
- Publishing of draft tags needs to remain restricted.

**Comment by alext** [comment]
Just looking at that screenshot, the child topics should be ordered by name.  They were on the previous index page that's now been removed.

**Comment by tijmenb** [comment]
@alext done

File: app/views/shared/tags/show.html.erb [hunk 1]
```diff
@@ -10,7 +10,14 @@
   </h1>
 
   <nav class="actions">
-    <%= link_to 'Edit', edit_polymorphic_path(@resource) %>
+    <% if gds_editor? %>
+      <%= link_to 'Edit', edit_polymorphic_path(@resource) %>
+    <% end %>
+
+    <% if @resource.is_a?(Topic) %>
+      <%= link_to 'Edit list', topic_lists_path(@resource) %>
```
Comment (alext) [inline app/views/shared/tags/show.html.erb]: This should not be shown for top-level topics, only for subtopics.

**Comment by alext** [comment]
One small outstanding thing, otherwise looks good to me.

**Pull request merged**